### DISCUSSION TITLE
New supported model: S100103, Lay-Z Helsinki

### DISCUSSION THREAD

**Discussion by hubaksis** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#170] [Show and tell]
Module S100103, Lay-Z Helsinki is working.
It's 6 wire, I used 510 Ohm resistors (instead of suggested 560 Ohms resistors between LLC and display (CLK, DATA, CS), as
reported by cyberfly79).
The module is connected to MQTT/Openhab. I don't use web interface at all.
The only noticed issue for now - sometimes when I switch the temperature, it sets 20 instead of the value I chose. I faced the issue only a couple of times, so hard to reproduce.

![IMG_20211229_165832](https://user-images.githubusercontent.com/13341477/148152306-e0c583ca-c0ce-40a3-bce7-f6286723d199.jpg)
![IMG_20211229_170057](https://user-images.githubusercontent.com/13341477/148152319-34e9f440-ad0b-4acf-ba60-7a0cd3b3def3.jpg)
![IMG_20211231_160031](https://user-images.githubusercontent.com/13341477/148152322-ea4b4884-3d99-4272-8eb0-1ed8266ff6e4.jpg)
![IMG_20211231_160035](https://user-images.githubusercontent.com/13341477/148152328-660dfa89-f186-44e4-ab2d-1cc8a4fff54b.jpg)
![IMG_20211231_160045](https://user-images.githubusercontent.com/13341477/148152333-c39a9da3-a698-4cd9-89ee-5061793455c1.jpg)

**Comment by kgreene1983** [comment]
how did you mount this to the unit?

**Reply by hubaksis** [reply]
I put it in the box, used a hot glue gun to make the box waterproof and put it inside the unit. 
Used some wires to attach it to the mounted elements.
I don't have a photo where exactly, but somewhere here (see arrows on the picture).
I decided not to install it near the heater just in case if it's too hot outside.
![image](https://user-images.githubusercontent.com/13341477/152623302-52c344a0-8b76-47b4-9c3e-b4b2ccc64ea2.png)

**Reply by kgreene1983** [reply]
Thanks for uploading.

I was really disappointed with the normal control system on it and thought about doing something with a wemos d1. Couldn't believe it when I came across this.. You would think for the small cost lazy spa would do it anyway

**Reply by hubaksis** [reply]
There is a WiFi option in these units, and the cost is $100 more. Not sure what are the features though.

**Reply by visualapproach** [reply]
@hubaksis the development branch now has a new method of setting the temperature. Should be more reliable but a little bit slower to get to the new target temperature.

**Reply by kgreene1983** [reply]
Just wondering why you need to add the resistor?

**Reply by hubaksis** [reply]
As per discussion here: https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/issues/54
Also https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/blob/master/Build-instructions-Bestway-WiFi-remote.pdf at the third page "Might need 560Ohm resistors between LLC and display (CLK, DATA, CS) as reported by cyberfly79" 
I haven't checked without them, just used them straight away.

**Reply by kgreene1983** [reply]
thanks for the info.

I'm using the PCB, could someone tell what pins (6 wire) I need to add the resistors too?
